jsonsaver/jsonsaver_test.go

```diff
@@ -0,0 +1,230 @@
+						FileContributor:   []string{"Protecode Inc.", "SPDX Technical Team Members", "Open Logic Inc.", "Source Auditor Inc.", "Black Duck Software In.c"},
+						FileDependencies:  []string{"SPDXRef-JenaLib", "SPDXRef-CommonsLangSrc"},
+						FileName:          "./src/org/spdx/parser/DOAPProject.java",
+						FileType:          []string{"SOURCE"},
+						LicenseConcluded:  "Apache-2.0",
+						LicenseInfoInFile: []string{"Apache-2.0"},
+					},
+				},
+				PackageHomePage:                     "http://ftp.gnu.org/gnu/glibc",
+				PackageLicenseComments:              "The license for this project changed with the release of version x.y.  The version of the project included here post-dates the license change.",
+				PackageLicenseConcluded:             "(LGPL-2.0-only OR LicenseRef-3)",
+				PackageLicenseDeclared:              "(LGPL-2.0-only AND LicenseRef-3)",
+				PackageLicenseInfoFromFiles:         []string{"GPL-2.0-only", "LicenseRef-2", "LicenseRef-1"},
+				PackageName:                         "glibc",
+				PackageOriginatorOrganization:       "ExampleCodeInspect (contact@example.com)",
+				PackageFileName:                     "glibc-2.11.1.tar.gz",
+				PackageVerificationCodeExcludedFile: "./package.spdx",
+				PackageVerificationCode:             "d6a770ba38583ed4bb4525bd96e50461655d2758",
+				PackageSourceInfo:                   "uses glibc-2_11-branch from git://sourceware.org/git/glibc.git.",
+				PackageSummary:                      "GNU C library.",
+				PackageSupplierPerson:               "Jane Doe (jane.doe@example.com)",
+				PackageVersion:                      "2.11.1",
+			},
+		},
+		UnpackagedFiles: map[spdx.ElementID]*spdx.File2_2{
+			"File": {
+				FileSPDXIdentifier: "File",
+				FileChecksums: map[spdx.ChecksumAlgorithm]spdx.Checksum{
+					"SHA1": {
+						Algorithm: "SHA1",
+						Value:     "d6a770ba38583ed4bb4525bd96e50461655d2758",
+					},
+					"MD5": {
+						Algorithm: "MD5",
+						Value:     "624c1abb3664f4b35547e7c73864ad24",
+					},
+				},
+				FileComment:       "The concluded license was taken from the package level that the file was .\nThis information was found in the COPYING.txt file in the xyz directory.",
+				FileCopyrightText: "Copyright 2008-2010 John Smith",
+				FileContributor:   []string{"The Regents of the University of California", "Modified by Paul Mundt lethal@linux-sh.org", "IBM Corporation"},
+				FileName:          "./package/foo.c",
+				FileType:          []string{"SOURCE"},
+				LicenseComments:   "The concluded license was taken from the package level that the file was .\nThis information was found in the COPYING.txt file in the xyz directory.",
+				LicenseConcluded:  "(LGPL-2.0-only OR LicenseRef-2)",
+				LicenseInfoInFile: []string{"GPL-2.0-only", "LicenseRef-2"},
+				FileNotice:        "Copyright (c) 2001 Aaron Lehmann aaroni@vitelus.",
+			},
+		},
+	}
+	tests := []struct {
+		name    string
+		args    args
+		wantW   string
```

Comment:
Is `wantW` being used here? I see that `wantErr` is being assigned and checked below, but it looks like this isn't actually checking to see whether the output of `Save2_2` matches what's expected.

That might be intentional, since we can't be certain about the exact output of bytes from the JSON marshaller. And I believe you're checking the actual contents of the saver output in the other tests in the subdirectory. If that's the case, though, then perhaps it makes sense to remove `wantW` here to avoid confusion?